jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/codeAction/CodeActionHandler.java

```diff
@@ -140,6 +142,10 @@
                         codeActions.addAll(ResourceMethodMultipleEntityParamsQuickFix.getCodeActions(context,
                                 diagnostic, monitor));
                     }
+                    if (diagnostic.getCode().getLeft().equals(Jax_RSConstants.DIAGNOSTIC_CODE_NO_PUBLIC_CONSTRUCTORS)) { // shaunak working here
+                        codeActions.addAll(ResourceMethodMultipleEntityParamsQuickFix.getCodeActions(context,
```

Comment:
As i understand, this code adds a quick fix to codeActions if the diagnostic code is DIAGNOSTIC_CODE_NO_PUBLIC_CONSTRUCTORS

Reply:
Changing line 146 to:
```java
codeActions.addAll(NoResourcePublicConstructorQuickFix.getCodeActions(context,
                                diagnostic, monitor));
```
and the quick fix shows up on the diagnostic for me: 
![image](https://user-images.githubusercontent.com/26146482/143302380-d20e94f6-736d-4bc2-b3ae-3cf1b02336b0.png)